kahuna/public/js/edits/image-editor.html

```diff
@@ -98,6 +98,13 @@
                     <button class="image-info__edit" ng:click="ctrl.showUsageRights = !ctrl.showUsageRights">
                         <gr-icon>edit</gr-icon>
                     </button>
+                    <button
+                        class="job-editor__apply-to-all"
+                        title="Apply these usage rights to all your current uploads"
+                        type="button"
+                        ng:if="ctrl.withBatch"
+                        ng:click="ctrl.batchApplyUsageRights()"
+                        >⇔</button>
```

Comment:
Could we swap this out for the batch apply all icon for labels while we're here?